trento/xml/sso-integration.xml

```diff
@@ -0,0 +1,1012 @@
+-e OIDC_CLIENT_SECRET=&lt;OIDC_CLIENT_SECRET&gt; \
+-e OIDC_BASE_URL=&lt;OIDC_BASE_URL&gt; \
+
+# Optional:
+-e OIDC_CALLBACK_URL=&lt;OIDC_CALLBACK_URL&gt; \
+
+...[other settings]...
+</programlisting>
+        </listitem>
+      </orderedlist>
+    </section>
+    <section xml:id="available-variables-for-openid-connect">
+      <title>Available variables for OpenID Connect</title>
+      <variablelist>
+        <varlistentry>
+          <term>
+            OIDC_CLIENT_ID
+          </term>
+          <listitem>
+            <para>
+              OIDC client id
+            </para>
+          </listitem>
+        </varlistentry>
+        <varlistentry>
+          <term>
+            OIDC_CLIENT_SECRET
+          </term>
+          <listitem>
+            <para>
+              OIDC client secret
+            </para>
+          </listitem>
+        </varlistentry>
+        <varlistentry>
+          <term>
+            OIDC_BASE_URL
+          </term>
+          <listitem>
+            <para>
+              OIDC base url
+            </para>
+          </listitem>
+        </varlistentry>
+        <varlistentry>
+          <term>
+            OIDC_CALLBACK_URL
+          </term>
+          <listitem>
+            <para>
+              OIDC callback url where the IDP is redirecting once the
+              authentication is completed (default value:
+              <uri>https://#{TRENTO_WEB_ORIGIN}/auth/oidc_callback</uri>)
+            </para>
+          </listitem>
+        </varlistentry>
+      </variablelist>
+    </section>
+  </section>
+  <section xml:id="using-oauth-20">
+    <title>Using OAuth 2.0</title>
+    <para>
+      Trento integrates with an IDP that uses the OAuth 2 protocol to
+      authenticate users accessing the Trento web console.
+    </para>
+    <para>
+      By default, OAuth 2.0 is disabled. You can enable OIDC when using
+      RPM packages or using Docker images.
+    </para>
+    <section xml:id="enabling-oauth-20-when-using-kubernetes-deployment">
+      <title>Enabling OAuth 2.0 when using kubernetes deployment</title>
+      <para>
+        To enable OAuth 2.0 when using kubernetes deployment with helm,
+        proceed as follows:
+      </para>
+      <orderedlist numeration="arabic">
+        <listitem>
+          <para>
+            Add the following variables to the previously documented
+            helm installation command:
+          </para>
+          <programlisting>
+HELM_EXPERIMENTAL_OCI=1 helm ... \
+   --set trento-web.oauth2.enabled=true \
+   --set trento-web.oauth2.clientId=&lt;OAUTH2_CLIENT_ID&gt; \
+   --set trento-web.oauth2.clientSecret=&lt;OAUTH2_CLIENT_SECRET&gt; \
+   --set trento-web.oauth2.baseUrl=&lt;OAUTH2_BASE_URL&gt; \
+   --set trento-web.oauth2.authorizeUrl=&lt;OAUTH2_AUTHORIZE_URL&gt; \
+   --set trento-web.oauth2.tokenUrl=&lt;OAUTH2_TOKEN_URL&gt; \
+   --set trento-web.oauth2.userUrl=&lt;OAUTH2_USER_URL&gt; \
+   --set trento-web.oauth2.scopes=&lt;OAUTH2_SCOPES&gt;
```

Comment:
The MD file describes the scopes parameter as optional:
"trento-web.oauth2.scopes</option> variable is optional with `profile email` as default value"
Maybe we should do it here as well?

Reply:
The `.md` and `.xml` files should have the exact same content. You don't need to review both.
Just have a look in the final `.xml`. If there is something wrong there, I will make changes in the `.md` file which are (should be at least, if I don't forget) translated to the final result